metrics-and-checks/checks-ensuring-correctness.md

```diff
@@ -9,7 +9,7 @@ db = client.integration('aqueduct_demo')
 customers = db.sql('SELECT * FROM customers;')
 
 @check(severity=CheckSeverity.ERROR)
-def non_negative_workflows(customers: pd.DataFrame) -> boolean:
+def non_negative_workflows(customers: pd.DataFrame) -> bool:
```

Comment:
I feel like these bug fixes are more urgent, can we break them out in a separate PR and get those merged in faster?